nixnet_examples/programmatic_database_api.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import absolute_import
```

Comment:
I couldn't find an equivalent example for what I'm trying to do. There is a C example called XNET_Database_Browser but it doesn't leverage add or remove alias. It might be good for the get database list or get database list size functions that I'm writing.

I trying to demonstrate that the same read/write code can be reused with different database aliases (different cluster, baud rates, etc.). This sounded like a common use case for aliases based on my conversation with Jeff.

How about `dynamic_database_usage.py` or `programatic_database_usage.py`?